src/index.html

```diff
@@ -11,7 +11,151 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset name="Personal information" >
+        <legend>Personal information:</legend>
+        <label for="surname">Surname:</label>
+        <input
+          type="text"
+          id="surname"
+          required
+          autocomplete="off"
+          class="input"
+        >
+        <br>
+        <label for="name">Name:</label>
+        <input
+          type="text"
+          id="name"
+          autocomplete="off"
+          class="input"
+        >
+        <br>
+        <label for="age">How old are You?</label>
+        <input
+          type="number"
+          id="age"
+          value="12"
+          min="1"
+          max="100"
+          class="input"
+        >
+        <br>
+        <label for="birth-date" >Full date of birth:</label>
+        <input
+          type="date"
+          id="birth-date"
```

Comment:
Use camelCase for id.
```suggestion
          id="birthDate"
```